packages/salesforcedx-vscode-replay-debugger/package.nls.json

```diff
@@ -0,0 +1,11 @@
+{
+  "launch_snippet_label_text": "Apex Replay Debugger: Launch",
+  "launch_snippet_description_text":
+    "A new configuration for launching Apex Replay Debugger",
+  "launch_snippet_name": "Launch Apex Replay Debugger",
+  "logfile_text": "Path to an Apex Debug Log file in the workspace folder.",
```

Comment:
Do we want to say what the file extension should be?

Reply:
(*.log) would be good to include.

Reply:
I can add that to suggest it's a .log file, but the code is not enforcing that extension because the user can rename their log file to anything. Is that okay?